CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Consistent probability sampler implementation. (#1379)
+- Add README to `go.opentelemetry.io/contrib/propagators/aws/xray`. (#2321)
```

Comment:
I wouldn't have added a changelog entry for a readme addition/change. It's not technically something meaningful for folks who upgrade from a previous version.

Reply:
Noted, that's a good point. Removed it.